bot/exts/pride/drag_queen_name.py

```diff
@@ -27,6 +28,6 @@ async def dragname(self, ctx: commands.Context) -> None:
         await ctx.send(random.choice(self.names))
 
```

Comment:
Doesn't let me comment there, but change L22 to be of the same format as everywhere else; 
```py
with Path("bot/resources/pride/drag_queen_names.json").open("r", encoding="utf8") as f:
```